src/components/Form/PinInput/hooks/usePinInput.ts

```diff
@@ -6,19 +6,42 @@ import { Keys } from 'helpers/accessibility';
 import { clamp } from 'helpers/math';
 import { useCustomEnsuredControl } from 'hooks/useEnsureControl';
 
+export const BiometricType = {
+  FACEID: 'faceid',
+  FINGERPRINT: 'fingerprint',
+} as const;
+
+// eslint-disable-next-line @typescript-eslint/no-redeclare
```

Comment:
We use disabling of eslint in this project in rare and very specific cases. Let's follow repository guidelines